build.cmd

```diff
@@ -353,6 +353,7 @@ if "%InstallPythonPackages%" == "True" (
     :: Run azureml-dataprep tests only in pyhon 3.7 as its an optional dependency
     if %PythonVersion% == 3.7 ( call "%PythonExe%" -m pip install --upgrade "azureml-dataprep>=1.1.12" )
     call "%PythonExe%" -m pip install --upgrade "%__currentScriptDir%target\%WheelFile%"
+    if %PythonVersion% == 3.5 ( call "%PythonExe%" -m pip install --upgrade pip )
```

Comment:
I'm seeing the error installing scikit on Windows Py27 as well in my PR CI checks. Is anyone else seeing this as well?

If so, should we add this to line 351 instead?

Reply:
good point, could u pls work with Mustafa and show him the repro

---
In reply to: [331269871](https://github.com/microsoft/NimbusML/pull/297#discussion_r331269871) [](ancestors = 331269871)

Reply:
I just had the issue this morning trying to create a Python 3.6 release build. Maybe this should be added for all python versions?

---
In reply to: [331608183](https://github.com/microsoft/NimbusML/pull/297#discussion_r331608183) [](ancestors = 331608183,331269871)